src/Tribe/Assets.php

```diff
@@ -165,7 +165,8 @@ public function enqueue( $forcibly_enqueue = null ) {
 			// If we have a set of conditionals we loop on then and get if they are true
 			foreach ( $asset->conditionals as $conditional ) {
 				$enqueue = call_user_func( $conditional );
-				if ( $enqueue ) {
+				// this may return false or null or? Only enqueue if they all pass by returning a truthy value
+				if ( empty( $enqueue ) ) {
```

Comment:
Possibly a case of `tribe_is_truthy`?

Reply:
Man, always forget that one! YES.

Also - we might discuss adding a way to implement AND/OR in here?